src/dispatch/static/dispatch/src/incident/ResourcesTab.vue

```diff
@@ -80,5 +115,17 @@ export default {
       "selected.conversation",
     ]),
   },
+
+  async mounted() {
+    this.ticketPluginEnabled = await this.isPluginEnabled("ticket")
+    this.conferencePluginEnabled = await this.isPluginEnabled("conference")
+    this.conversationPluginEnabled = await this.isPluginEnabled("conversation")
+    this.storagePluginEnabled = await this.isPluginEnabled("storage")
+    this.documentPluginEnabled = await this.isPluginEnabled("document")
```

Comment:
Same comment as above with cases.

Reply:
I'm planning to add a feature that allows users to select which resource to add.
For example, only creating a storage resource.

Reply:
Done.